ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -653,6 +653,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             modelBuilder.Entity<DrugResistanceProfile>(entity =>
             {
+                entity.Property(e => e.Species)
+                    .HasMaxLength(30)
+                    .HasDefaultValue("No result");
+                entity.Property(e => e.DrugResistanceProfileString)
+                    .HasMaxLength(30)
+                    .HasDefaultValue("No result");
```

Comment:
Do you think it would be worth adding a test for this? It looks like it should be a pretty easy test to write and although there's not really any logic here it would potentially be useful as a way of recording a business requirement.

Reply:
It probably would be worth adding a test but I'm not too sure on where to put it. Currently it looks like the only place the notification creation step is tested is in the UI tests. There are plenty integration tests for editing a pre-existing draft in all of the different edit pages, but no tests for adding the draft in the first place. 

This feels like a bit of an oversight, but again probably out of scope for this ticket. To test this properly care will need to be taken wrt to user identity and access rights: the draft needs to be created in the right PHEC/TB service else the user will get redirected to an "Access denied" page and not have the rights to change the notification location to then mean they have access rights again. I can raise a follow-on ticket if that's ok with you?

Reply:
I think the right place to add this would be in the integration tests. It's true that you would have to create a new test class (`CreateNotificationTests`) but the extra setup involved in this test _should_ be minimal as there's no pre-existing data required. I don't think you need to get as far as assigning the draft to a TB Service, since the logic that we're interested in happens before that.

Reply:
But if you want to raise a follow-on ticket for this then I am ok with that